### PR TITLE
[Release] sdk/resourcemanager/devcenter/armdevcenter/0.1.0

### PR DESCRIPTION
**Pull request by Alancere**
https://github.com/Azure/sdk-release-request/issues/3030

### CODE REVIEW

**Review by tadelesh — APPROVED** [review]

File: sdk/resourcemanager/devcenter/armdevcenter/build.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+// This file enables 'go generate' to regenerate this specific SDK
+//go:generate pwsh.exe ../../../../eng/scripts/build.ps1 -skipBuild -cleanGenerated -format -tidy -generate resourcemanager/devcenter/armdevcenter
```
Comment (tadelesh) [inline sdk/resourcemanager/devcenter/armdevcenter/build.go:5]:
```suggestion
//go:generate pwsh ../../../../eng/scripts/build.ps1 -skipBuild -cleanGenerated -format -tidy -generate resourcemanager/devcenter/armdevcenter
```

Comment (Alancere) [reply sdk/resourcemanager/devcenter/armdevcenter/build.go]: removed

**Review by tadelesh — APPROVED** [review]

**Pull request merged**